_includes/tldr.html

```diff
@@ -0,0 +1,6 @@
+<div class="tldr">
+  <div class="title">{{ site.data.lang.text.tldr }}</div>
```

Comment:
I propose a simple change to allow for a custom title:
```suggestion
  <div class="title">{{ include.title | default: site.data.lang.text.tldr }}</div>
```